### ISSUE TITLE
Recently added notifications 

### ISSUE THREAD

**Issue by pax0707** [Tautulli/Tautulli#692]
Happening only for recently added. Playback notifications work fine. 
Notifications are missing the 1st part of the message.
Started happening after fix for #691. 

This is the template:

```
<movie>{title} ({year}) [{studio}]</movie><tv>{show_name} - S{season_num00}E{episode_num00} - {episode_name} [{studio}]</tv><music>{artist_name} - {album_name} - {episode_num00} - {track_name}</music> added to {server_name} - {library_name}. 

Poster: {poster_url} 
<movie>IMDB: {imdb_url}</movie><tv>TVDB: {thetvdb_url}</tv><music>Last.FM: {lastfm_url}</music>
Trakt: {trakt_url}
Genders: {genres}
Content rating: {content_rating}
Summary: {summary}
Tagline: {tagline}
Rating: {rating}
```

This is notification being sent:

```
Timestamp   Agent   Action  Subject Text    Body Text   Script Args
2016-05-07 08:11:40 Telegram    created PlexPy (APOLLO) - recently added    TVDB: https://thetvdb.com/?tab=series&id=310802
Trakt: https://trakt.tv/search/tvdb/310802?id_type=show
Genders: Comedy
Content rating: 
Summary: Ronny is a Malaysian student who’s come to Australia to study law. He’s smart, driven and competitive, which would make him the perfect law student, if he actually wanted to be a lawyer. Blunt, barbed and not afraid to cut through the BS, it’s not long before he is at odds with most people on campus; not to mention his mum. Ronny finds himself caught between his old life and his new life, between the international students and the local students, and between his mum’s expectations and his own dreams, whatever the hell they may be.
Tagline: 
Rating: 
2016-05-07 08:11:38 E-mail  created PlexPy (APOLLO) - recently added    TVDB: https://thetvdb.com/?tab=series&id=310802
Trakt: https://trakt.tv/search/tvdb/310802?id_type=show
Genders: Comedy
Content rating: 
Summary: Ronny is a Malaysian student who’s come to Australia to study law. He’s smart, driven and competitive, which would make him the perfect law student, if he actually wanted to be a lawyer. Blunt, barbed and not afraid to cut through the BS, it’s not long before he is at odds with most people on campus; not to mention his mum. Ronny finds himself caught between his old life and his new life, between the international students and the local students, and between his mum’s expectations and his own dreams, whatever the hell they may be.
Tagline: 
Rating:
```

**Comment by JonnyWong16** [comment]
Fixed a3a62b1d94fa03c9610c2acd501035f40884597e (I hope).
Update and try again.

**Comment by pax0707** [comment]
Working now.
